modules/nw-proxy-configure-object.adoc

```diff
@@ -102,23 +96,16 @@ spec:
 ----
 +
 --
-<1> A proxy URL to use for creating HTTP connections outside the cluster. The
-URL scheme must be `http`.
-<2> A proxy URL to use for creating HTTPS connections outside the cluster.
-<3> A comma-separated list of destination domain names, domains, IP addresses or
-other network CIDRs to exclude proxying.
+<1> A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be `http`.
+<2> A proxy URL to use for creating HTTPS connections outside the cluster. The URL scheme must be either `http` or `https`. Specify a URL for the proxy that supports the URL scheme. For example, most proxies will report an error if configured with `https` but only supports `http`. The failure message may not propagate to the logs, however, and appear to be a network connection failure. If using a proxy that listens for `https` connections from the cluster, you may need to configure the cluster to accept the Certificate Authorities and certificates used by the proxy.
```

Comment:
To me, this is getting long enough that it might require breaking out beyond a callout. Judgment call, though.

```suggestion
<2> A proxy URL to use for creating HTTPS connections outside the cluster. The URL scheme must be either `http` or `https`. Specify a URL for the proxy that supports the URL scheme. For example, most proxies will report an error if they are configured to use `https` if they only support `http`. This failure message may not propagate to the logs, however, and can appear to be a network connection failure instead. If using a proxy that listens for `https` connections from the cluster, you may need to configure the cluster to accept the certificate authorities and certificates that the proxy uses.
```